components/layout/blog-view.hbs

```diff
@@ -0,0 +1,226 @@
+                        <time class="content-block__date-blog" datetime="2021-06-10"><span
+                                class="content-block__date-month-day-year" aria-label="June">06.02.21</span></time>
+                    </div>
+                    <h4 class="content-block__title anchor-primary">
+                        Fix your talent cycle or lose your talent.
+                    </h4>
+                    <p class="content-block__text-blog">The talent cycle is broken, and fixing it absolutely must be
+                        a top priority for leaders.</p>
+                    <div class="blog-tags">
+                        <p class="content-block__text-blog-tag">Career Development, Experience Design, Strategy,
+                            Skill Development</p>
+                    </div>
+                </div><!-- .content-block -->
+            </a><!-- .content-block--clickable -->
+        </div><!-- .blog-content -->
+
+        <div class="blog-content">
+            <a href="#" class="content-block--clickable">
+                <div class="content-block">
+                    <img class="content-block__image" src="https://www.fillmurray.com/g/545/747"
+                        role="presentation" alt="" />
+                </div><!-- .content-block -->
+                <div class="content-block">
+                    <div class="blog-author-date">
+                        <label for="source" class="content-block__author">By Susie Hall</label>
+                        <time class="content-block__date-blog" datetime="2021-06-10"><span
+                                class="content-block__date-month-day-year"
+                                aria-label="June">06.02.21</span></time>
+                    </div>
+                    <h4 class="content-block__title anchor-primary">
+                        What every creative team needs to be successful.
+                    </h4>
+                    <p class="content-block__text-blog">Learn how the role of the Creative Operations leader
+                        creates success for an in-house design team.</p>
+                    <div class="blog-tags">
+                        <p class="content-block__text-blog-tag">Career Development, Experience Design, Strategy,
+                            Skill Development</p>
+                    </div>
+                </div><!-- .content-block -->
+            </a><!-- .content-block--clickable -->
+        </div><!-- .blog-content -->
+
+    </div> <!-- .two-col-content -->
+
+
+    <div class="grid-col-content__three">
+
+        <div class="blog-content">
+            <a href="#" class="content-block--clickable">
+                <div class="content-block">
+                    <img class="content-block__image" src="https://www.fillmurray.com/g/268/160"
+                        role="presentation" alt="" />
+                </div><!-- .content-block -->
```

Comment:
Same comment about content-blocks inside content-blocks. But also, I think we'll want to mark this up as a <picture>+<srcset> to get the right setup for the image scaling.